frontend/server/src/main/java/com/amazonaws/ml/mms/http/ControlRequestHandler.java

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package com.amazonaws.ml.mms.http;
+
+import com.amazonaws.ml.mms.archive.InvalidModelException;
+import com.amazonaws.ml.mms.archive.Manifest;
+import com.amazonaws.ml.mms.archive.ModelArchive;
+import com.amazonaws.ml.mms.common.ErrorCodes;
+import com.amazonaws.ml.mms.openapi.OpenApiUtils;
+import com.amazonaws.ml.mms.util.NettyUtils;
+import com.amazonaws.ml.mms.wlm.ModelManager;
+import com.amazonaws.ml.mms.wlm.WorkerInitializationException;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.handler.codec.http.FullHttpRequest;
+import io.netty.handler.codec.http.HttpMethod;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.netty.handler.codec.http.QueryStringDecoder;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Function;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A class handling inbound HTTP requests to the control API.
+ *
+ * <p>This class
+ */
+public class ControlRequestHandler extends HttpRequestHandler {
```

Comment:
Can the name instead be ControlPlaneRequestHandler ?

Reply:
Changed